main.py

```diff
@@ -0,0 +1,26 @@
+from random import randint
+import math
+N = input("Enter amount of numbers: ")
+array = list()
+a = input("Enter first number: ")

```

Comment:
Pairs should be created automatically
Array: 1, 2, 3, 4
Pairs: 
1, 2 
1, 3
1, 4
2, 3
2 ,4
3, 4